components/omega/doc/devGuide/CondaEnv.md

```diff
@@ -7,45 +7,43 @@ code, you will need your own
 [conda](https://conda.io/projects/conda/en/latest/index.html) environment with
 the necessary packages.
 
-(omega-dev-install-mambaforge)=
+(omega-dev-install-miniforge3)=
 
-## Installing Mambaforge
+## Installing Miniforge3
 
 If you have not already done so, you will install
-[Mambaforge](https://github.com/conda-forge/miniforge#mambaforge), preferably
+[Miniforge3](https://github.com/conda-forge/miniforge#miniforge3), preferably
 somewhere in your home directory but possibly somewhere in your scratch space.
```

Comment:
Maybe add a caveat about a home directory install, i.e., it may cause you to run out of space over time since quotas are typically small there.